packages/ses/package.json

```diff
@@ -22,20 +22,15 @@
   "type": "module",
   "types": "./index.d.ts",
   "main": "./dist/ses.cjs",
-  "module": "./ses.js",
+  "module": "./index.js",
   "browser": "./dist/ses.umd.js",
   "unpkg": "./dist/ses.umd.js",
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "import": "./ses.js",
+      "import": "./index.js",
       "require": "./dist/ses.cjs",
       "browser": "./dist/ses.umd.js"
-    },
-    "./lockdown": {
```

Comment:
Why not just make this an alias for `./ses.js` et al?  That would provide trivial backwards-compatibility.